src/snowflake/snowpark/query_history.py

```diff
@@ -14,7 +14,10 @@ class QueryRecord(NamedTuple):
 
 
 class QueryHistory:
-    """A context manager that listens to and records SQL queries that are pushed down to the Snowflake database."""
+    """A context manager that listens to and records SQL queries that are pushed down to the Snowflake database.
+
+    Refer to :meth:`snowflake.snowpark.Session.query_history`.
```

Comment:
I feel it'd better to use `See Also:`

Reply:
I think the original way is fine also. I don't have a strong opinion either way.